news/9455.feature.rst

```diff
@@ -0,0 +1,2 @@
+New resolver: The order of dependencies resolution has been tweaked to traverse
+the dependency graph in a more depth-first approach.
```

Comment:
Thank you for working on this!

Should this say "breadth first approach" rather than "depth first approach"?
From reading the PR description and code comments it seems the new approach focuses on shallower deps first?